src/jit/hwintrinsiccodegenxarch.cpp

```diff
@@ -2128,6 +2131,8 @@ void CodeGen::genBMI1OrBMI2Intrinsic(GenTreeHWIntrinsic* node)
         case NI_BMI2_MultiplyNoFlags:
         case NI_BMI2_X64_MultiplyNoFlags:
         {
+            // These do not support containment
+            assert(!op2->isContained());
```

Comment:
Many of these are just temporary, correct? That is we should be able to make them containable eventually and change `emitIns_AR_R` into `emitInsStoreInd` (in a separate PR)?

Reply:
Yes, it should be possible, but I didn't want to tackle that complexity at this point.